src/components/Banner/Banner.tsx

```diff
@@ -31,40 +41,82 @@ export default function Banner({ queryRef, text, requireAuth = false }: Props) {
 
   const isAuthenticated = Boolean(query.viewer?.user?.id);
 
-  const [dismissed, setDismissed] = usePersistedState(
-    JAN_18_MIGRATION_BANNER_DISMISSED_LOCAL_STORAGE_KEY,
-    false
-  );
+  const [dismissed, setDismissed] = usePersistedState(LOCAL_STORAGE_KEY || '', false);
 
   const hideBanner = useCallback(() => {
     setDismissed(true);
   }, [setDismissed]);
 
   return dismissed || text.length === 0 || (requireAuth && !isAuthenticated) ? null : (
-    <StyledBanner>
-      <StyledClose onClick={hideBanner}>&#x2715;</StyledClose>
-      <BaseM color={colors.white}>{text}</BaseM>
-    </StyledBanner>
+    <StyledContainer>
+      <StyledBanner>
+        <StyledContent>
+          {title && <StyledTitle>{title}</StyledTitle>}
+          <StyledText color={colors.offBlack}>{text}</StyledText>
+        </StyledContent>
+        <StyledAction>
+          <div onClick={hideBanner}>{children}</div>
```

Comment:
instead of disabling the banner when they click "Sign Poster" in the banner, let's wait to disable it until they click the "Sign Poster" button on the actual page (that opens Figma)

that way the user still has a way of coming back to this page if they decide to do it later

Reply:
such a good idea

Reply:
a simple fix would be to just remove the hideBanner handler from the action component, and simply have the user dismiss it by clicking the "X"

```js
        <StyledAction>
          {actionComponent}
          <StyledClose onClick={hideBanner}>&#x2715;</StyledClose>
        </StyledAction>
```

Reply:
@Robinnnnn Great idea! Seems like a simple fix. Will implement this way first. 

I've been thinking on how to close the `banner` from outside the `banner` component itself 🤔 . Seems like over-engineer the component lol

Reply:
oh yeah... we'd need some global context like the ToastProvider haha